int8 done
string group_name
int64 neighbor_count
float32 cost_average
moveit_msgs/RobotState max_cost_neighbor

